xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -96,6 +96,7 @@ glusterd_handle_friend_req(rpcsvc_request_t *req, uuid_t uuid, char *hostname,
     char rhost[UNIX_PATH_MAX + 1] = {0};
     dict_t *dict = NULL;
     dict_t *peer_ver = NULL;
+    int totcount = 5;  // Total count of specific keys in array
```

Comment:
It's better to not hardcode this:

```suggestion
    int totcount = sizeof(specific_key_suffix) / sizeof(specific_key_suffix[0]);
```